### DISCUSSION TITLE
from file manager open sqlitefile with sqlitestudio, option to keep it permanent

### DISCUSSION THREAD

**Discussion by marcobra** [pawelsalawa/sqlitestudio#4454] [Q&A] [answered]
### Details

from file manager, when i choose to open a .sqlite file "Open with SQLstudio" i cannot get the option to made it permanent in SQLitestudio

Please add an option to keep as permanent, not only when we open a file .sqlite from menu
I suggest to add this option to the right menu showed by click sqlite name on left panel.

Thank

**Comment by pawelsalawa** [comment]
SQLiteStudio has nothing to to with it. It's up to the file manager.

**Comment by marcobra** [comment]
i don't talk of file manager i'm talking of SQLitestudio option to keep the opened file as permanent

**Comment by pawelsalawa** [comment, marked as answer]
Ohhh. I misunderstood you.

When you open such file, then in SQLiteStudio right-click on that database, pick "Edit database" and there mark the "Permanent" checkbox and confirm with "Ok".

**Comment by marcobra** [comment]
ok i found it is under the right click menu "edit the database" and there is the option to keep it as permanent

Sorry 

Thanks
